optical/converter/utils.py

```diff
@@ -198,3 +203,106 @@ def get_id_to_class_map(df: pd.DataFrame):
     """
     set_df = df.drop_duplicates(subset="class_id")[["category", "class_id"]]
     return set_df.set_index("class_id")["category"].to_dict()
+
```

Comment:
If you want to skip docstring for a function which is small and obvious, it is acceptable. But sphinx will still try to generate a docstring for it nonetheless. Please add a single `_` before the function name to tell sphinx to exclude those functions from `autofunction`.

Reply:
Thanks added _before some small function's